core/assets/bundles/bundle_fr.properties

```
@@ -1877,11 +1877,11 @@ gz.turrets = Recherchez et placez 2 \uf861 [accent]Duos[] pour défendre votre n
 gz.duoammo = Rechargez vos Duos avec du [accent]cuivre[], en utilisant les convoyeurs.
 gz.walls = Les [accent]Murs[] peuvent empêcher les attaques ennemies d'atteindre vos constructions.\nPlacez des \uf8ae [accent]murs de cuivre[] autour de vos tourelles.
 gz.defend = Ennemis en approche, préparez-vous à défendre.
-gz.aa = Les unités aériennes ne peuvent pas être facilement repoussées avec des tourelles standard.\n\uf860 Les [accent]Disperseurs[] sont d'excellentes tourelles anti-aériennes, mais requierent du \uf837 [accent]plomb[] en tant que munition.
-gz.scatterammo = Approvisionnez le Disperseur avec du [accent]plomb[], en utilisant des convoyeurs.
+gz.aa = Les unités aériennes ne peuvent pas être facilement repoussées avec des tourelles standard.\n\uf860 Les [accent]Disperseurs[] sont d'excellentes tourelles anti-aériennes, mais requiert du \uf837 [accent]plomb[] en tant que munition.
```

```suggestion
gz.aa = Les unités aériennes ne peuvent pas être facilement repoussées avec des tourelles standard.\n\uf860 Les [accent]Disperseurs[] sont d'excellentes tourelles anti-aériennes, mais requièrent du \uf837 [accent]plomb[] en tant que munition.
```
On parle des disperseurs en général, pas d'un seul en particulier.